<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewAlarmDialog</class>
 <widget class="QDialog" name="QNewAlarmDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="titleEdit">
        <property name="placeholderText">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QAlarmTimeValueButton" name="timeButton"/>
      </item>
      <item>
       <widget class="QAlarmDateValueButton" name="dateButton"/>
      </item>
      <item>
       <widget class="QAlarmDaysValueButton" name="daysButton"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QAlarmSoundValueButton" name="soundButton" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="landscapeButtonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="portraitButtonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAlarmTimeValueButton</class>
   <extends>QPushButton</extends>
   <header>qalarmtimevaluebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QAlarmDateValueButton</class>
   <extends>QPushButton</extends>
   <header>qalarmdatevaluebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QAlarmDaysValueButton</class>
   <extends>QPushButton</extends>
   <header>qalarmdaysvaluebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QAlarmSoundValueButton</class>
   <extends>QWidget</extends>
   <header>qalarmsoundvaluebutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>landscapeButtonBox</tabstop>
  <tabstop>portraitButtonBox</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>timeButton</tabstop>
  <tabstop>dateButton</tabstop>
  <tabstop>daysButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
